xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -54,6 +74,11 @@ viewers.Comments = Class.create({
     this.addCancelListener();
     this.addEditListener();
     this.addPreview(this.form);
+    this.reloadEditor({
+      callback: function () {
+        this.addPreview(this.form);
```

Comment:
I don't understand this callback: we don't want the addPreview for WYSIWYG editor and if I understand properly the reloadEditor is only useful for WYSIWYG editor...

Reply:
I read back reloadEditor and I got it wrong: it's used for any editor. Now I guess there is a problem in it since we need to call explicitely addPreview.